monkey/monkey_island/cc/services/config.py

```diff
@@ -68,6 +69,12 @@ def get_config(is_initial_config=False, should_decrypt=True, is_island=False):
             ConfigService.decrypt_config(config)
         if not is_island:
             config.get("cnc", {}).pop("aws_config", None)
+            # strip out credentials
+            for field in {"exploit_user_list", "exploit_password_list"}:
+                config.get("credentials", {}).pop(field, None)
+            for cred_type in {"exploit_lm_hash_list", "exploit_ntlm_hash_list", "exploit_ssh_keys"}:
+                config.get("internal", {}).get("exploits", {}).pop(field, None)
+
```

Comment:
It looks like you added this code back in. Was that accidental?

Reply:
Looks like this is the first commit in this branch. Maybe Shreya pushed it by accident? I'll revert